zulip_bots/zulip_bots/bots/ping/doc.md

```diff
@@ -0,0 +1,29 @@
+# Ping Bot
+
+This bot allows users to check availability of a given website or IP
+address across 10 random locations using the free API provided by
+[check-host.net](https://check-host.net). The API documentation can be
+found [here](https://check-host.net/about/api).
```

Comment:
This doc is supposed to be displayed in the /integrations page of a Zulip server, so it might be read by folks who just want a running bot and don't care about the API, given that a user doesn't need to access it and retrieve a key when running this bot.